# putout-plugin-replace-const-assignment

[NPMIMGURL]: https://img.shields.io/npm/v/putout-plugin-.svg?style=flat&longCache=true
[NPMURL]: https://npmjs.org/package/putout-plugin-"npm"

🐊[**Putout**](https://github.com/coderaiser/putout) plugin adds abilitity to replace const assignment.

## install

```sh
npm i putout putout-plugin-replace-const-assignment -D
```

## Rule

```json
{
    "plugins": [
        "replace-const-assignment"
    ]
}
```

## ❌ Example of incorrect code

```js
```

## ✅ Example of correct code

```js
```

## License

MIT
