# putout-plugin-evaluate-expression

[NPMIMGURL]: https://img.shields.io/npm/v/putout-plugin-.svg?style=flat&longCache=true
[NPMURL]: https://npmjs.org/package/putout-plugin-"npm"

ğŸŠ[**Putout**](https://github.com/coderaiser/putout) plugin adds abilitity to evaluate expressions.

## install

```sh
npm i putout putout-plugin-evaluate-expression -D
```

## Rule

```json
{
    "plugins": [
        "evaluate-expression"
    ]
}
```

## âŒ Example of incorrect code

```js
```

## âœ… Example of correct code

```js
```

## License

MIT
